metadata/energy/oil_gas_wells.md

```diff
@@ -0,0 +1,77 @@
+# Title
+
+Utah Oil and Gas Wells
+
+## ID
+
+303a22e9-0dc0-4e5e-9053-d671321900e9
+
+## Brief Summary
+
+Surface location of oil and gas wells from the Department of Natural Resources, Oil, Gas, and Mining Division.
+
+## Summary
+
+This dataset contains point locations of oil and gas wells from the Oil, Gas, and Mining Division (DOGM) at the Utah Department of Natural Resources. Points contain attributes such as the API code, well and company name, and well type and status.
+
+## Description
+
+### What is the dataset?
+
+Oil and gas wells in Utah are managed by the Utah Department of Natural Resources [Oil, Gas and Mining Division](https://ogm.utah.gov/). This dataset contains all oil and gas wells in Utah, including those being actively drilled, still in the permit process, and those that have been abandoned.
+
+### What is the purpose of the dataset?
+
+This dataset provides the opportunity for the Division to collect more descriptive down-hole information for each wellbore permitted and drilled.
+
+<!--- This is from the original metadata. Do we still see this dataset as an opportunity to collect more descriptive information, or does this dataset have a different purpose now? --->
+
+### What does the dataset represent?
+
+Points in this dataset include the API code, well and company name, account number, filed number, field name, elevation, locations coordinates, lease numbers, well type and status, total cumulative oil, gas and water, and many other attributes.
+
+### How was the dataset created?
+
+This data was developed by contractors sponsored in part by the U.S. Department of Energy and the Ground Water Protection Council.
+
+<!--- What else might the user need to know about how this dataset was created? Do we maintain the dataset now, or is maintained by someone else? --->
+
+### How reliable and accurate is the dataset?
+
+Initially, much of the down-hole information will not be available because there was no place to save it in the previous databases. Eventually, as new wells are drilled and old legacy data is entered, more descriptive data will become available for the wells in Utah.
```

Comment:
This is incorrect. Down-hole information is a [separate dataset](https://opendata.gis.utah.gov/datasets/utahDNR::oil-and-gas-down-hole-locations/about).

Yes, this is the most complete and current dataset for wells. It's actively maintained by DOGM.